src/app/entities/courses/components/course-item/course-item.component.scss

```diff
@@ -1,62 +1,77 @@
 @import "../../../../../mixins";
 @import "../../../../../colors";
 
-.course-item-container {
+.course-item {
   @include card-style;
   width: 100%;
   margin-bottom: 24px;
   padding: 24px;
   box-sizing: border-box;
+  border: solid 4px transparent;
 
-  .course-item-header {
-    position: relative;
+  &--top-rated {
+    background-color: $yellow-light;
+  }
+
+  .course-item__header {
```

Comment:
you can use `&__header { ... }` to shorten class names

Reply:
Yes, good catch